src/Microsoft.DocAsCode.Dfm/MarkdownValidators/MarkdownValidationRule.cs

```diff
@@ -9,8 +9,14 @@ namespace Microsoft.DocAsCode.Dfm.MarkdownValidators
 
     public class MarkdownValidationRule
     {
+        /// <summary>
+        /// The contract name of rule.
+        /// </summary>
         [JsonProperty("name", Required = Required.Always)]
```

Comment:
I understand we need to be compatible, but it seems to be confusing after we have an "id" for the rule. How about add a property "contractName" and mark "name" as obsolete?

---
In reply to: [73273456](https://github.com/dotnet/docfx/pull/545#discussion_r73273456) [](ancestors = 73273456,73125280)